python/ray/train/v2/_internal/state/export.py

```diff
@@ -86,8 +92,33 @@ def train_run_attempt_to_proto(attempt: TrainRunAttempt) -> ProtoTrainRunAttempt
     return proto_attempt
 
 
+def _to_proto_dashboard_panel(panel: Panel) -> ProtoTrainRun.DashboardPanel:
+    """Convert Dashboard Panel to protobuf format."""
+    proto_panel = ProtoTrainRun.DashboardPanel(
+        id=panel.id,
+        title=panel.title,
+    )
```

Comment:
Do we need more information, like description?

Reply:
You can add description to the panel definitions themselves. not necessary here.

Reply:
You can add description to the panel definitions themselves. not necessary here.

Reply:
Oh okay what about the title?

Reply:
title is useful still because we show title while the embeds are loading.